indra/llcommon/llstring.cpp

```diff
@@ -545,29 +545,29 @@ std::string utf8str_truncate(const std::string& utf8str, const S32 max_len)
 
 std::string utf8str_symbol_truncate(const std::string& utf8str, const S32 symbol_len)
 {
-    if (0 == symbol_len)
```

Comment:
Our coding standard prefers spaces over tabs and specifically warns not to do unnesesary convertions.
https://wiki.secondlife.com/wiki/Coding_standard